### DISCUSSION TITLE
Documentation of the pipes

### DISCUSSION THREAD

**Discussion by YummyToadies** [frankframework/frankframework#2856] [Q&A] [answered]
Good day, where can I find good documentation about the framework?

At the readthedocs manual (https://frank-manual.readthedocs.io/en/latest/) I can't find proper documentation which pipelines are available and what properties one have and how to build up an adapter (except for the example).

Thanks in advance.

**Comment by jacodg** [comment, marked as answer]
Sorry for the late reply. You can find all the elements and attributes and their documentation in the Frank!Doc::

https://frankdoc.frankframework.org/#!/All

The Frank!Doc is also available for code completion in your IDE when you reference the FrankConfig.xsd in your configuration xml:

https://github.com/ibissource/frank-runner#code-completion-with-frankconfigxsd

Don't hesitate to ask more questions. You can also contact us at https://wearefrank.nl/contact/
